relay-dynamic-config/src/defaults.rs

```diff
@@ -622,6 +622,102 @@ fn span_metrics(
                     .always(), // already guarded by condition on metric
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/mobile.slow_frames@none".into(),
```

Comment:
Unless you need percentiles, a gauge metric should be enough here. We're transitioning to gauges for the other metrics as well (see https://github.com/getsentry/relay/pull/3448).
```suggestion
            mri: "g:spans/mobile.slow_frames@none".into(),
```